bootstrap/roles/ocp4-post-acs/tasks/post_ci.yaml

```diff
@@ -47,6 +47,39 @@
         namespace: "{{ ocp4_demo_pipeline_namespace }}"
       type: Opaque
 
+# - name: Get Service Account objects
```

Comment:
I've fixed (check the main branch) grabbing directly the secret token itself, instead of grepping for the sa pipeline token. Shall we move to the main branch proposal and avoid to use the shell commands?